src/tools/cleaner.js

```diff
@@ -43,11 +43,34 @@ const recurseReplaceBank = (obj, bank = {}) => {
     if (typeof out !== 'string') {
       return out;
     }
+
     Object.keys(bank).forEach(key => {
       // Escape characters (ex. {{foo}} => \\{\\{foo\\}\\} )
-      const s = String(key).replace(/[-[\]/{}()\\*+?.^$|]/g, '\\$&');
-      const re = new RegExp(s, 'g');
-      out = out.replace(re, bank[key]);
+      const escapedKey = String(key).replace(/[-[\]/{}()\\*+?.^$|]/g, '\\$&');
```

Comment:
Changed the names here to be a bit more descriptive. I _think_ we could drop the `String` constructor. Since `key` is derived from `Object.keys`, we should be able to safely assume it is always a string.